extensions/wikia/ExactTargetUpdates/tasks/ExactTargetWikiTask.class.php

```diff
@@ -0,0 +1,117 @@
+<?php
+namespace Wikia\ExactTarget;
+
+use Wikia\Tasks\Tasks\BaseTask;
+use Wikia\Util\Assert;
+
+class ExactTargetWikiTask extends BaseTask {
+
+	private $client;
+
+	const STATUS_OK = 'OK';
+
+	public function __construct( $client = null ) {
+		$this->client = $client;
+	}
+
+	/**
+	 * Update wiki or create if doesn't exist
+	 * @param int $wikiId
+	 * @return string
+	 * @throws \Wikia\Util\AssertionException
+	 */
+	public function updateWiki( $wikiId ) {
+		Assert::true( !empty( $wikiId ), 'Wiki ID missing' );
+
+		$client = $this->getClient();
+
+		/* Update or create Wiki in external service */
+		$client->updateWiki( $wikiId, $this->getWikiData( $wikiId ) );
+
+		return self::STATUS_OK;
+	}
+
+	/**
+	 * Delete wiki and categories mapping
+	 * @param int $wikiId
+	 * @return string
+	 * @throws \Wikia\Util\AssertionException
+	 */
+	public function deleteWiki( $wikiId ) {
+		Assert::true( !empty( $wikiId ), 'Wiki ID missing' );
+
+		$client = $this->getClient();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "$client" which has the same name as the field declared at line 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS1117)